Automattic/notifications-panel
# Mentions: various bugfixes

This PR fixes these bugs on the mentions mixin:

* #170 enter key to select active mention.
* Adds a space after the username to improve writing flow.
* moves suggestion fetching to `componenDidMount` to remove delay after first mention.
* suggestions now position correctly depending on the available space on the screen.
* removes flash of suggestion list when repositioning.
* when using arrow shortcuts, the list reposition itself to show hidden elements correctly.
* some minor cleanups in the code.
* ~~bonus: `package-lock.json`.~~

How to test:

Just use add mentions on the reply field of any comment. The list should appear almost instantaneously, selection with both mouse and keyboard should work as expected and the list should position itself either above or below the reply field depending on the available space on the page.
